### PR TITLE
fix(DB/Creature): update Shandy's creature_text

### PR DESCRIPTION
**Pull request by YggdrasilWotLK**
Makes Shandy say his for a Suitable Disguise in the correct order

## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [X] Database (SAI, creatures, etc).
Order of dialogues was mixed up, causing issues such as the quest seeming like it had failed (ie. Sandy said the failure dialogue) despite the fact that there's no scripted failure event in the underlying cpp script, or asking for shirts instead of saying the second half of the scripted intro RP

## Issues Addressed:

- Closes 
https://github.com/azerothcore/azerothcore-wotlk/issues/744

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**
https://www.youtube.com/watch?v=DHmeKq9t5Z0

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [x] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1. .quest add 24556 (horde) or .quest add 20438 (ally)
2. .go creature 1790
3. initiate the quest RP and see the dialogue mismatch

## Known Issues and TODO List:

- [x] The failure event is missing

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by YggdrasilWotLK** [comment]
New PR after I accidentally closed https://github.com/azerothcore/azerothcore-wotlk/pull/19690 @Helias  :)

**Comment by TheSCREWEDSoftware** [comment]
I don't know what i was supposed to do as i felt the instructions of the npc werent matching the action. Testing this PR video yotubue [here](https://www.youtube.com/watch?v=AJj_Z8zryew&ab_channel=TheSCREWEDSoftware)

File: data/sql/updates/pending_db_world/shandy-fix.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
+DELETE FROM `creature_text`
+WHERE `CreatureID` = 36856
+  AND `GroupID` IN (0, 1, 2, 3, 4, 5, 6, 7, 8, 9);
```
[the remark this answers is not in the document]
Comment (YggdrasilWotLK) [reply data/sql/updates/pending_db_world/shandy-fix.sql:4]: I can add the ID of course but it's not necessary as the table is arranged on stock AC :)

**Comment by YggdrasilWotLK** [comment]
> I don't know what i was supposed to do as i felt the instructions of the npc werent matching the action. Testing this PR video yotubue [here](https://www.youtube.com/watch?v=AJj_Z8zryew&ab_channel=TheSCREWEDSoftware)

Doesn't seem like you imported the SQLs correctly as the Magister confirmation menu should not exist if they've been run. I just noticed I had forgotten to add the Arcanite's confirmation menu so I've added that too. Please let me know if you need any help importing the commited file :)

**Comment by TheSCREWEDSoftware** [comment]
I downloaded this commit I pressumed the worldserver would update the db to any pending ones. Whenever i get a chance to manually import the SQL i will test it again.

**Comment by TheSCREWEDSoftware** [comment]
I tested with the last changes fetched (from 2 hours ago) and manually ran the SQL.

New test on Youtube: [here](https://www.youtube.com/watch?v=HLWwKiTFsZU&ab_channel=TheSCREWEDSoftware)

**Review by Helias — APPROVED** [review]

**Comment by elthehablo** [comment]
Thanks for the PR. For future reference, please stick to the PR naming conventions for archivation purposes (easy look-up).

**Pull request merged**